### ISSUE TITLE
Timestamp conversion is wrong

### ISSUE THREAD

**Issue by joannesalfa** [brockpetrie/vue-moment#66]
`<p>{{ 1516004936125 | moment("dddd, MMMM Do YYYY") }}</p>`

It returned Wednesday, April 14th 50010

It's supposed to be:
Monday, January 15th 2018

Check this https://www.epochconverter.com with this timestamp number

**Comment by BrockReece** [comment]
That's odd, using `$moment(1516004936125) | moment("dddd, MMMM Do YYYY")` seems to work though.

https://codesandbox.io/s/znl03op4y4

**Comment by joannesalfa** [comment]
Nice, would you mind to update your docs to avoid confusion?

**Comment by BrockReece** [comment]
Sure, but I might spend a bit more time investigating the underlying issue, as the code I sent may be considered as an anti pattern.

**Comment by brockpetrie** [comment]
@joannesalfa Our conversion isn't "wrong," there's just differing opinions about how a unix epoch should be represented. Standard unix timestamp is traditionally given in seconds, not milliseconds like a JS timestamp. Special-casing this is extremely complex, so we have to choose one or the other when inferring the case.

I'm willing to revisit this, but it does a bit of complexity.

For now, you can skirt this by passing milliseconds like so: `new Date(mymilliseconds)`

**Comment by joannesalfa** [comment]
@brockpetrie This example is from API of Firebase, `firebase.database.ServerValue.TIMESTAMP` I cannot set it by passing milliseconds.

**Comment by brockpetrie** [comment]
@joannesalfa You aren't able to pass your millisecond variable like so?:

```html
<p>{{ new Date(1516004936125) | moment("dddd, MMMM Do YYYY") }}</p>
```

**Comment by brockpetrie** [comment]
I think for 4.0.0 I'll mimic EpochConverter's way of doing this. If input consists of fewer than or equal to 11 digits, assume it's a timestamp given in seconds. If input is greater than 11 digits, assume it's given in ms.

**Comment by brockpetrie** [comment]
This is improved in #69. Will close this when that's merged in via `4.0.0`.

**Comment by joannesalfa** [comment]
Yes, `new Date(timestamp)` works after all. 

Nice to read about enhancement.
